### PR TITLE
Import mutations.

### PR DESCRIPTION
**Pull request by nicolasnoble**

### CODE REVIEW

**Comment by nicolasnoble** [comment]
@y-zeng: you'll need this first.

File: test/cpp/interop/BUILD [hunk 1]
```diff
@@ -64,12 +89,57 @@ cc_library(
...
+        "stress_test.cc",
+        "stress_interop_client.cc",
+        "stress_interop_client.h",
+    ],
+    deps = [
+        ":client_helper_lib",
+        "//:grpc++",
+        "//test/cpp/util:metrics_server_lib",
+        "//test/cpp/util:test_config",
+    name = "interop_client",
```
Comment (y-zeng) [inline test/cpp/interop/BUILD]: Something is missing here?

**Comment by ctiller** [comment]
Is this still needed?

**Comment by nicolasnoble** [comment]
@ctiller: yes.

**Comment by ctiller** [comment]
Please address @y-zeng's comment

**Comment by nicolasnoble** [comment]
Yep, done.

**Comment by nicolasnoble** [comment]
All green, changed requested applied, merging.

**Pull request merged**